burnman/seismic.py

```diff
@@ -427,7 +473,8 @@ class STW105(SeismicTable):
 
     def __init__(self):
         SeismicTable.__init__(self)
-        table = tools.read_table("input_seismic/STW105.txt") # radius, pressure, density, v_p, v_s
+        # radius, pressure, density, v_p, v_s
+        table = tools.read_table("input_seismic/STW105.txt")
```

Comment:
I'd quite like to see read_table be deprecated by np.loadtxt or np.genfromtxt - I wrote it before I knew much python, and I think it would be better to use standard (and more powerful) tools where possible.